code/optimize/src/main/java/com/adobe/marketing/mobile/optimize/Optimize.java

```diff
@@ -327,4 +327,77 @@ private static void failWithError(final AdobeCallback<?> callback, final AdobeEr
             callbackWithError.fail(error);
         }
     }
+
+    public static void displayPropositions(List<OptimizeProposition> propositions) {
```

Comment:
`displayPropositions` makes it seem like this API displays the propositions. Can you rename it to `trackDisplayedPropositions` or simple `displayedPropositions`

Reply:
done @spoorthipujariadobe